src/projects.html

```diff
@@ -123,8 +124,8 @@ <h2 class="section-heading">Pimp My Cause</h2>
           <div class="project-info">
             <h2 class="section-heading">IPPR</h2>
             <h3> Express, CartoDb, Leaflet, ejs, jQuery</h3>
-            <p>This project is aiming to improve transparency around oil contracts. For that, we are building a website using NodeJS/Express on the backend and JavaScript on the front end. We are keeping the data on CartoDB, which includes a PostGIS database on the cloud and some data manipulation and analysis tools. We also use LeafLet, which is a JavaScript framework to plot the CartoDB data on a map. We have currently reached our first milestone and work is still on going.
-            </p>
+            <p>Namibia's Institute for Public Policy Research (IPPR) is an organisation with a mission to deliver independent, analytical, and critical research into social, political and economic issues that affect development in Namibia.</p>
```

Comment:
we can go with the less sensitive text - but we do have  a screenshot that gives away the name of the country etc...